02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <title>Khadijah's Bio</title>
 </head>
 <body>
-  <h1>Indian Restaurant</h1>
+  <h1><u>Khadijah's Bio</u></h1>
+  <a ref="">I own a company called Best Lead Generation that
+    specializes in providing <div> business-to-business lead generation
+    services to companies in order to help <div>grow their business.</a>
+
+    <h3><u>Things I like:</u><h3>
+    <ul>
+    <li> Reading
+    <li> Hiking
+    <li> Baking
+    <li> Cooking
+    <li> Coding
+    </ul>
+
+<h3><u>Things I don't like:</u><h3>
+    <ol>
+    <li> Bananas
+    <li> Bad traffic
+    <li> Bad weather
+    </ol>
```

Comment:
The lists you have on your page are a good example of what I mean by adding indents to your HTML code. In the ordered list example, you would open your `<ol>` tag, go to the next line, indent and then start your list items (`<li>` tags). When closing your ordered list tag, make sure it lines up with the opening ordered list tag.